modules/base/src/components/options/Option.vue

```diff
@@ -0,0 +1,350 @@
+<template>
+  <div class="row" style="margin-top: 0px;">
+    <div class=" column" style="text-align: left;">
+      <h5>
+        <span
+          v-if="record.ExpertiseLevel == 1"
+          data-tooltip="Changing this setting may impair or break Portmaster functionality."
+          data-position="right center"
+        >
+          <i class="ui yellow exclamation icon"></i>
+        </span>
+        <span
+          v-if="record.ExpertiseLevel == 2"
+          data-tooltip="Changing this setting may have a disastrous impact on Portmaster. Only change if you really know what you are doing."
+          data-position="right center"
+        >
+          <i class="ui red radiation icon"></i>
+        </span>
+        {{ record.Name }}
+        <span
+          v-if="record.RequiresRestart"
+          data-tooltip="Requires (manually) restarting the Portmaster in order to take effect."
+          data-position="right center"
+        >
+          <i class="ui blue redo icon"></i>
+        </span>
+        <div
+          v-if="record.ReleaseLevel == 1"
+          data-tooltip="This feature/setting is in beta and changing it might not work as expected."
+          class="ui yellow tiny label"
+        >
+          Beta
+        </div>
+        <div
+          v-if="record.ReleaseLevel == 2"
+          data-tooltip="This feature/setting is experimental and changing it might not work as expected or have an unforeseen, potentially disastrous impact."
+          class="ui red tiny label"
+        >
+          Experimental
+        </div>
+      </h5>
+      {{ record.Description }}
+      <div v-if="record.Help" v-on:click="showHelp = !showHelp" class="ui small button">
+        Input Help
+      </div>
+      <!-- TODO: replace with interactive version -->
+      <div v-else-if="rKey == 'config:filter/lists'" v-on:click="showHelp = !showHelp" class="ui small button">
+        Show Available Filter Lists
+      </div>
+    </div>
+
+    <div v-bind:class="[activeState() ? 'active' : 'inactive', 'column']">
+      <i v-bind:class="[activeState() ? 'green' : 'grey outline', 'circle icon']"></i>
+
+      <!-- Status -->
+      <span style="width: 10px; margin-right: 10px;">
+        <i class="big green check icon" v-if="successState"></i>
+        <i class="big red x icon" v-else-if="errorState"></i>
+      </span>
+
+      <!-- Value -->
+      <div class="ui active inverted dimmer" v-if="loadingState">
+        <div class="ui text loader">Loading</div>
+      </div>
+
+      <OptionBoolean
+        v-if="record.OptType == optTypeBool"
+        :record="record"
+        :successState="successState"
+        :errorState="errorState"
+      ></OptionBoolean>
+      <OptionIntSecurityLevel
+        v-else-if="record.ExternalOptType == 'security level'"
```

Comment:
I guess we should also move those `security level`, ... stuff into constants as they appear a lot (also in the go part of the application)

Reply:
Have not yet figured out how to do constants in a nice way with Vue.js.